### PR TITLE
Update values.yaml

### PR DESCRIPTION
**Pull request by pratmit**
CG is renamed as FG

### CODE REVIEW

File: src/harness/values.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ global:
   saml:
     autoaccept: false
   license:
-    cg: ''
```
Comment (arya-harness) [inline src/harness/values.yaml:45]: these would require multiple changes inside sub-charts of helm-platform where global.cg usages were already there.
cc: @kapilgarg1996

Comment (kapilgarg1996) [reply src/harness/values.yaml:45]: @pratmit is it something being changed everywhere across harness ?

Comment (pratmit) [reply src/harness/values.yaml:45]: Yes, CG is renamed to FG across Harness.

**Comment by puthrayaharness** [comment]
Closed PR as this is getting tracked for Dev effectiveness

**Pull request closed without merging**